src/main/java/com/thejavatar/pokemongogo/PokemonDecorator.java

```diff
@@ -92,14 +95,72 @@ public int getEvolutionsPossible() {
         	return 0;
     }
     
+    public String getPokeball() {
+        if (ItemId.ITEM_MASTER_BALL == pokemon.getPokeball())
```

Comment:
We should stick to the convention that is already used in the other part of the codes, that is: always use `{`.